docs/src/languages/java/migration.md

```diff
@@ -0,0 +1,166 @@
+
+{{< /note >}}
+
+## Microservices
+
+You have the option to use several languages in microservices. If you're using Java there are several options to aggregate these services into a microservices:
+
+* [Maven Modules](https://maven.apache.org/guides/mini/guide-multiple-modules.html)
+* [Gradle Multi-project](https://guides.gradle.org/creating-multi-project-builds/)
+* [Git submodules](https://docs.platform.sh/development/submodules.html)
+
+[Platform.sh supports multiple applications](https://docs.platform.sh/configuration/app/multi-app.html) and there are two options:
+
+* [One application YAML file to each application](https://docs.platform.sh/configuration/app.html)
+* [Aggregate all applications in a single file with applications.yaml](https://docs.platform.sh/configuration/app/multi-app.html#applicationsyaml)
+
+| Article                                                      | Content                                                      |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [Microservices in the cloud, part two](https://platform.sh/blog/2019/microservices-in-the-cloud-part-two/) | [Source](https://github.com/EventosJEspanol/latin-america-micro-profile) |
+| [Microservices in the cloud, part one](https://platform.sh/blog/2019/microservices-in-the-cloud-part-one/) | [Source](https://github.com/EventosJEspanol/latin-america-micro-profile) |
+| [Multiple Applications](https://community.platform.sh/t/multiple-applications-tomcat/468) | [Source](https://github.com/platformsh-examples/tomcat-multi-app) |
+| [Configure multi-applications with applications.yaml](https://community.platform.sh/t/how-to-configure-multi-applications-with-applications-yaml/552) | [Source](https://github.com/platformsh-examples/tomcat-multi-app-applications) |
+
+{{< note >}}
+
+As a single application, you have the option to set load balancer to some or [all applications in the project cluster](https://community.platform.sh/t/how-to-configure-load-balancer-in-a-multiple-applications/554).
+
+{{< /note >}}
+
+## Access to services included at Platform.sh
+
+[Unlike other PaaS services, Platform.sh is batteries included, which means that you don’t need to subscribe to an external service to get a cache or a search engine](https://docs.platform.sh/configuration/services.html). However, you can use a database or any services such as a transition process, just be aware of the [firewall](https://docs.platform.sh/configuration/app/firewall.html). 
+
+When applications need to access a service, it is important to activate the [Relationships tag](https://docs.platform.sh/configuration/app/relationships.html), because. by default an application may not talk to any other container within a project it includes others projects as a microservices architecture.
```

Comment:
```suggestion
When applications need to access a service, it is important to include the [Relationships key](https://docs.platform.sh/configuration/app/relationships.html), because. by default an application may not talk to any other container within a project it includes others projects as a microservices architecture.
```